scripts/_common.sh

```diff
@@ -7,57 +7,18 @@
 # dependencies used by the app
 YNH_PHP_VERSION="7.3"
 
-pkg_dependencies="postgresql php${YNH_PHP_VERSION}-tidy php${YNH_PHP_VERSION}-intl php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-pgsql php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-curl"
+pkg_dependencies="php${YNH_PHP_VERSION}-tidy php${YNH_PHP_VERSION}-intl php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-curl"
```

Comment:
don't we need to add `php${YNH_PHP_VERSION}-mysql`

Reply:
using the current yunohost php, it's already included, but we can add it